paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/activity/BasePaymentSheetActivity.kt

```diff
@@ -61,10 +61,16 @@ internal abstract class BasePaymentSheetActivity : AppCompatActivity() {
                     PaymentConfiguration.init(this, checkoutResponse.publishableKey)
 
                     onSuccess(
-                        PaymentSheet.CustomerConfiguration(
-                            id = checkoutResponse.customerId,
-                            ephemeralKeySecret = checkoutResponse.customerEphemeralKeySecret
-                        ),
+                        if (checkoutResponse.customerId != null
```

Comment:
This looks like duplicate code line 52 PlaygroundViewModel.

Reply:
Perhaps an extension function on CheckoutResponse that will return the CustomerConfiguration?

Reply:
We want to further simplify the custom and complete examples by using the [simpler endpoint](https://glitch.com/edit/#!/stripe-mobile-payment-sheet) that doesn't take any parameter, then this code will be deleted. I just didn't have time to do it yet. For now moved to CheckoutResponse.